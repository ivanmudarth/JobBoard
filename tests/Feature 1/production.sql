-- Feature 1
INSERT INTO User (username, password, name) VALUES ('user1', 'abcabc', 'John Doe')
INSERT INTO User (username, password, name) VALUES ('dhruvs', 'pass1', 'Dhruv Shah')
INSERT INTO User (username, password, name) VALUES ('ivanm', 'pass2', 'Ivan Mudarth')

SELECT * FROM User

SELECT ID FROM User WHERE username = 'user1' AND password = 'acbabc'
SELECT ID FROM User WHERE username = 'dhruvs' AND password = 'pass1'
SELECT ID FROM User WHERE username = 'ivanm' AND password = 'pass2'
