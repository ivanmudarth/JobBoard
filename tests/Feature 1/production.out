Execute:
> SELECT * FROM User

+ ------- + ------------- + ------------- + --------- + ---------- + ---------------- +
| id      | username      | password      | name      | email      | description      |
+ ------- + ------------- + ------------- + --------- + ---------- + ---------------- +
| 1       | user1         | abcabc        | John Doe  |            |                  |
| 2       | dhruvs        | pass1         | Dhruv Shah |            |                  |
| 3       | ivanm         | pass2         | Ivan Mudarth |            |                  |
| NULL    | NULL          | NULL          | NULL      | NULL       | NULL             |
+ ------- + ------------- + ------------- + --------- + ---------- + ---------------- +
4 rows


Execute:
> SELECT ID FROM User WHERE username = 'user1' AND password = 'abcabc'

+ ------- +
| ID      |
+ ------- +
| 1       |
+ ------- +


Execute:
> SELECT ID FROM User WHERE username = 'dhruvs' AND password = 'pass1'

+ ------- +
| ID      |
+ ------- +
| 2       |
+ ------- +


Execute:
> SELECT ID FROM User WHERE username = 'ivanm' AND password = 'pass2'

+ ------- +
| ID      |
+ ------- +
| 3       |
+ ------- +